src/staging/src/pages/active-advisories.js

```diff
@@ -161,10 +161,10 @@ const PublicActiveAdvisoriesPage = ({ data }) => {
 
     // exclude unpublished parks
     let q =
-      "/public-advisories/count?protectedAreas.published_at_null=false&protectedAreas.isDisplayed=true"
```

Comment:
instead of removing this line, can you comment it out and also add a code comment with a link to my comment from below...
https://github.com/bcgov/bcparks.ca/pull/505/files#r1067160153